docs/custom-config.md

```diff
@@ -69,3 +69,77 @@ product "consul" {
 Beginning with version `0.4.0`, `hcdiag` supports redactions. Redactions enable users to tell `hcdiag` about patterns of text that should be omitted from the results bundle.
 
 Read more about them [here](./redactions.md).
+
+## Custom Config Examples
+
+### Collect additional Consul information
+
+**Example:** we want a `-consul` run to collect additional information from several commands:
+
+* The Consul memberlist: `consul members`
+* Raft peer information: `consul operator raft list-peers`
+* A custom journalctl command that's different from the builtin: `journalctl -u consul --since=yesterday --output=json`
+* another custom command (in this case, just running `cat` to collect a file's contents into the bundle `Results.json` file with no redactions).
```

Comment:
Since the previous three bullet points start with capital letters and don't include ending punctuation, I think it would be worth doing the same here.
```suggestion
* Another custom command (in this case, just running `cat` to collect a file's contents into the bundle `Results.json` file with no redactions)
```